process/block/preprocess/rewardsHandler.go

```diff
@@ -495,3 +504,44 @@ func (rtxh *rewardsHandler) GetAllCurrentFinishedTxs() map[string]data.Transacti
 func (rtxh *rewardsHandler) IsInterfaceNil() bool {
 	return rtxh == nil
 }
+
+func (rtxh *rewardsHandler) displayCalculatedRewardTxs() {
+	for _, tx := range rtxh.rewardTxsForBlock {
+		log.Trace("rewardTxsForBlock",
```

Comment:
maybe add all the fields here? valid on all for loops

Reply:
Done